### PR TITLE
WIP: Refactor/ethTransferv2

### PR DESCRIPTION
**Pull request by gesseekur**
Update query to EthTransferV2
and add filter to address transfers

### CODE REVIEW

File: newclient/src/modules/address/handlers/AddressTransfers/transfers.graphql [hunk 1]
```diff
@@ -1,16 +1,30 @@
 fragment TransferSummary on Transfer {
     transactionHash
+    block
```
Comment (olgakup) [inline newclient/src/modules/address/handlers/AddressTransfers/transfers.graphql:3]: Can you remove block and status from the fragment

File: newclient/src/modules/address/handlers/AddressTransfers/transfers.graphql [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
     name
     symbol
     decimals
 }
 
+query getEthTransfersV2($hash: String, $filter: TransferFilter, $_limit: Int, $_nextKey: String) {
+    getEthTransfersV2(owner: $hash, filter: $filter, limit: $_limit, nextKey: $_nextKey) {
+        transfers {
+            transfer {
+                ...TransferSummary
```
Comment (olgakup) [inline newclient/src/modules/address/handlers/AddressTransfers/transfers.graphql:20]: You can put this back like that: 
```
transfer {
      ....TransferSummary
      block
      status
}
```

Comment (gesseekur) [reply newclient/src/modules/address/handlers/AddressTransfers/transfers.graphql:18]: added!

File: newclient/src/modules/address/handlers/AddressTransfers/transfers.graphql (hunk 1 above)
Comment (gesseekur) [reply newclient/src/modules/address/handlers/AddressTransfers/transfers.graphql]: removed!

**Pull request merged**